scripts/bug_classifier.py

```diff
@@ -22,9 +23,12 @@ def classify_bugs(model_name, classifier):
     else:
         model_file_name = f"{model_name}model"
 
-    assert os.path.exists(
-        model_file_name
-    ), f"{model_file_name} does not exist. Train the model with trainer.py first."
+    if not os.path.exists(model_file_name):
+        print(f"{model_file_name} does not exist. Downloading the model....")
```

Comment:
Nit: use logger.info (you'll also need to setup the logger like other scripts do).